### PR TITLE
Fix/hotifx prod performance

### PR DESCRIPTION
**Pull request by dan1229**
Fresh off the presses - need to merge this to fix that high octane bug in that one instance. I promise this is good to go no need to check.

### CODE REVIEW

**Comment by oudeismetis** [comment]
seems legit

**Comment by oudeismetis** [comment]
seems legit

**Comment by whusterj** [comment]
Dabby Duck is the hero we deserve.

File: {{cookiecutter.project_slug}}/clients/vue3/src/views/Home.vue [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="home">
-    <img alt="Project Logo" src="../assets/logo.png" />
+    <img alt="Project Logo" src="../assets/logo.jpg" />
```
Comment (oudeismetis) [inline {{cookiecutter.project_slug}}/clients/vue3/src/views/Home.vue:3]: I'm sorry, on this project we only accept `.jpeg` files

Comment (oudeismetis) [inline {{cookiecutter.project_slug}}/clients/vue3/src/views/Home.vue:3]: I'm sorry, on this project we only accept `.jpeg` files

Comment (oudeismetis) [inline {{cookiecutter.project_slug}}/clients/vue3/src/views/Home.vue:3]: I'm sorry, on this project we only accept `.jpeg` files

**Pull request closed without merging**